ui/src/utils/defaultQueryConfig.js

```diff
@@ -10,8 +12,17 @@ export default function defaultQueryConfig(id) {
       time: null,
       tags: [],
     },
+    fill: NULL_STRING,
     areTagsAccepted: true,
     rawText: null,
     status: null,
   }
+
+  if (isKapacitorRule) {
+    delete queryConfig.fill
+  }
```

Comment:
Let's not modify this object in place plz :)

Reply:
Why's that?